src/index.ts

```diff
@@ -19,7 +19,9 @@ import { YourSheet } from '/path-to-your-sheet/your-sheet.ts'
 const MySheet = new Sheet(
   'MySheet',
   {
-    exampleField: TextField(),
+    firstName: TextField(),
+    lastName: TextField(),
+    email: TextField()
```

Comment:
I would avoid any email until we build a proper email field.  maybe age? as a NumberField instead?